src/app/components/bitcoin-fees-list/use-bitcoin-fees-data.ts

```diff
@@ -0,0 +1,191 @@
+import { useMemo } from 'react';
+
+import { Money, btcTxTimeMap } from '@leather.io/models';
+import {
+  type UtxoResponseItem,
+  useAverageBitcoinFeeRates,
+  useCryptoCurrencyMarketDataMeanAverage,
+} from '@leather.io/query';
+import {
+  baseCurrencyAmountInQuote,
+  capitalize,
+  createMoney,
+  formatMoneyPadded,
+  i18nFormatCurrency,
+  isUndefined,
+} from '@leather.io/utils';
+
+import type { TransferRecipient } from '@shared/models/form.model';
+
+import type { FeeDisplayInfo, FeeType } from '@app/common/fees/use-fees';
+import {
+  type DetermineUtxosForSpendArgs,
+  determineUtxosForSpend,
+  determineUtxosForSpendAll,
+} from '@app/common/transactions/bitcoin/coinselect/local-coin-selection';
+import { getSizeInfo } from '@app/common/transactions/bitcoin/utils';
+
+function getFeeForList(
+  determineUtxosForFeeArgs: DetermineUtxosForSpendArgs,
+  isSendingMax?: boolean
```

Comment:
violates this rule imo https://github.com/ryanmcdermott/clean-code-javascript?tab=readme-ov-file#dont-use-flags-as-function-parameters

Can it be two separate fns? When calling `getFeeForList(x, true)` it isn't clear what true does.